lightningd/remote_hsmd_inplace

```diff
@@ -0,0 +1 @@
+../../bin/remote_hsmd_inplace
```

Comment:
Maybe this is not intentional?

`remote_hsmd_inplace` is a (deprecated) binary at the moment.  We dropped support for it because it is not currently used and would be better implemented w/ a refactor of all the proxies.

We might need a separate PR to remove any cruft associated w/ it.

Reply:
yes was unintentional